core/templates/dev/head/galleries/gallery.html

```diff
@@ -21,7 +21,12 @@
   <script type="text/javascript">
     GLOBALS.LANGUAGE_CODES_AND_NAMES = JSON.parse(
       '{{LANGUAGE_CODES_AND_NAMES|js_string}}');
-    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse('{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse(
+      '{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    {% if explorations_list %}
```

Comment:
In the process of moving it all the front end. The search functionality is starting to become very unstable at the moment. I have a quiz tomorrow, so I'll get to it later on tomorrow.